src/main/java/org/junit/internal/runners/statements/RunBefores.java

```diff
@@ -20,6 +31,9 @@ public RunBefores(Statement next, List<FrameworkMethod> befores, Object target)
 
     @Override
     public void evaluate() throws Throwable {
+        if (null != previous) {
```

Comment:
If that's the case, the constructor should specify that the value can be `null`.